pyregion/parser_helper.py

```diff
@@ -167,22 +167,19 @@ def __init__(self, text):
         self.text = text
 
     def __repr__(self):
-        return "Property : " + repr(self.text)
+        return "%s : %s" % (type(self).__name__, repr(self.text))
```

Comment:
Use the more modern `str.format()`here like we do consistently in Astropy I think.